src/Protector.php

```diff
@@ -22,6 +22,7 @@
 use Illuminate\Support\Facades\Artisan;
 use Illuminate\Support\Facades\Http;
 use Illuminate\Support\Facades\Storage;
+use Laravel\Telescope\Telescope;
```

Comment:
Static analysis might flag this as an error, because the Telescope namespace is not part of the package dependencies.

Reply:
Removed the use statement and now the FQN is used instead.